Automattic/wp-calypso
# Domains: New domain status page design

This is a very rough implementation of the new domain status page design. It's feature flagged behind `domains/new-status-design` and currently doesn't implement all the possible statuses but is a good start. the idea is to merge this so we can split-up further work into smaller PRs

#### Changes proposed in this Pull Request

* Implement the basic domain status component
* feature flag all of it

#### Testing instructions

* enable `domains/new-status-design` (or run calypso locally) and check that the wpcom domain is showing the new domain status screen
* check that newly registered domains show the "Activating..." status in the first 30 minutes
* check that unverified contact info notice is shown for unverified contacts
* check that expired notice (and "Action required") is shown for expired domains
* some icons might not show properly if https://github.com/Automattic/wp-calypso/pull/39438 is not deployed by then

see: p99Zz8-UK-p2
depends on https://github.com/Automattic/wp-calypso/pull/39438 (a bunch of material icons)
